Modules/_functoolsmodule.c

```diff
@@ -39,6 +39,51 @@ get_functools_state(PyObject *module)
 }
 
 
+
+const char *getrepr(PyObject *obj) {
```

Comment:
You should make `getrepr()` and `gettype()` private:

```suggestion
const char *_Py_getrepr(PyObject *obj) {
```

Reply:
I left these accidentally, disregard.